File: src/server/Adfnet.Service/Models/MyProfileModel.cs
using System;
using System.Collections.Generic;
using Adfnet.Core.ValueObjects;

namespace Adfnet.Service.Models
{
    [Serializable]
    public class MyProfileModel
    {
        public UserModel UserModel { get; set; }
        public string Message { get; set; }
        public DateTime LastLoginTime { get; set; }
        public List<RootMenu> RootMenus { get; set; }
    }
}

File: src/client/web/private/src/app/models/my-profile-model.ts
import { UserModel } from './user-model';
import { RootMenu } from './root-menu';

export class MyProfileModel {
    userModel: UserModel;
    message: string;
    lastLoginTime: Date;
    rootMenus: RootMenu[];
    constructor() {
        this.userModel = new UserModel();
    }
}
